cmd/minishift/cmd/status.go

```diff
@@ -34,8 +34,9 @@ var statusCmd = &cobra.Command{
 	Short: "Gets the status of the local OpenShift cluster.",
 	Long:  `Gets the status of the local OpenShift cluster.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
+		api := libmachine.NewClient(constants.GetMinipath(), constants.MakeMiniPath("certs"))
```

Comment:
@gbraad You mean we should print profile name also in `minishift status` output? It means we have to change the put to stay consistent with and without profile.